app/src/main/java/com/lastaoutdoor/lasta/ui/screen/moreinfo/MoreInfoScreen.kt

```diff
@@ -456,7 +462,7 @@ fun ActivityPicture(activityToDisplay: Activity) {
       Image(
           painter = painterResource(id = defaultId),
           contentDescription = "Default Activity Picture",
-          modifier = Modifier.padding(5.dp).size(90.dp))
+          modifier = Modifier.padding(5.dp).size(90.dp).testTag(testTag))

```

Comment:
The activity icon looks rather big on my screen, did you try with smaller sizes to see how it goes ?